medusa/app.py

```diff
@@ -392,6 +392,15 @@ def __init__(self):
         self.TELEGRAM_ID = ''
         self.TELEGRAM_APIKEY = ''
 
+        self.USE_DISCORD = False
+        self.DISCORD_NOTIFY_ONSNATCH = False
+        self.DISCORD_NOTIFY_ONDOWNLOAD = False
+        self.DISCORD_NOTIFY_ONSUBTITLEDOWNLOAD = False
+        self.DISCORD_WEBHOOK = None
+        self.DISCORD_NAME = 'pymedusa'
+        self.DISCORD_AVATAR_URL = 'https://raw.githubusercontent.com/pymedusa/medusa/master/themes-default/slim/static/images/ico/favicon-144.png'
```

Comment:
```suggestion
        self.DISCORD_AVATAR_URL = '{base_url}/images/ico/favicon-144.png'.format(base_url=self.BASE_PYMEDUSA_URL)
```

Reply:
Hmm. `https://raw.githubusercontent.com` is something different then our cdn. Don't really feel much for creating a new constant specially for this?

Reply:
But it points to the same image?

Reply:
I tried it. But didn't work

Reply:
Strange.
https://cdn.pymedusa.com/images/ico/favicon-144.png
and
https://raw.githubusercontent.com/pymedusa/medusa/master/themes-default/slim/static/images/ico/favicon-144.png

both work for me.